src/Platform/Microsoft.Testing.Platform/Helpers/System/SystemConsole.cs

```diff
@@ -13,11 +13,19 @@ internal sealed class SystemConsole : IConsole
     /// <summary>
     /// Gets the height of the buffer area.
     /// </summary>
+    [UnsupportedOSPlatform("android")]
```

Comment:
We should probably have all these also on the `IConsole`.

Reply:
Yes, this is actually a false negative I reported in https://github.com/dotnet/roslyn-analyzers/issues/7579